src/aiori-S3-libs3.c

```diff
@@ -116,7 +119,7 @@ static void def_bucket_name(s3_options_t * o, char * out_name, char const * path
   }
   *out_name = '\0';
 
-  // S3Status S3_validate_bucket_name(const char *bucketName, S3UriStyle uriStyle);
+  S3Status S3_validate_bucket_name(const char *bucketName, S3UriStyle uriStyle);
```

Comment:
The function isn't called here. So it is just a declaration. Can you comment it out or use it?

Reply:
you're right! Since the different S3 providers impose different rules on the length of bucket names and their structure, i did comment it out at this stage.